pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -1003,6 +1003,12 @@ func publishZedAgentStatus(getconfigCtx *getconfigContext) {
 		ForceFallbackCounter:  ctx.forceFallbackCounter,
 		CurrentProfile:        getconfigCtx.currentProfile,
 		RadioSilence:          getconfigCtx.radioSilence,
+		DeviceState:           ctx.devState,
```

Comment:
Are we sure the fields in the context are initialized by this moment?
As far as I understand, these fields are initialized only in the `PublishDeviceInfoToZedClou` function. Are we sure the function is called before this one (`publishZedAgentStatus`)?

Reply:
devState can be zero (which I believe prints as "unknown") early in boot.